tests/mappers/test_axisymmetric_2d_to_3d.py

```diff
@@ -35,16 +36,24 @@ def test_instantiation(self):
     def test_initialize(self):
         mp_name_in = 'wall_in'
         mp_name_out = 'wall_out'
+        key = 'angle'
+        if key in self.parameters['settings']:
+            self.angle = self.parameters['settings']['angle']
+            print(key + ' is set on ' + str(self.angle) + ' degrees')
+        else:
+            self.angle = 360
+            print( key + ' is set on ' + str(self.angle) + ' degrees')
```

Comment:
Modify this to  `angle = self.parameters['settings'].get('angle', 360)` (I would not make angle a class variable, as you only use it locally)